renderer/src/components/FilAddressForm.tsx

```diff
@@ -11,6 +15,18 @@ interface FilAddressFormProps {
   enableEditMode: () => void;
 }
 
+const checkAddressString = (address: string) => {
+  if (address.startsWith('0x')) {
+    delegatedFromEthAddress(address)
+  } else if (address.startsWith('f4')) {
+    ethAddressFromDelegated(address)
+  } else if (address.startsWith('f1')) {
+    newFromString(address)
```

Comment:
Should we warn the user that sending funds to f1 addresses is more expensive because we have to go through the FilForwarder smart contract? Or is this a technical detail that we don't want to bother our non-technical users with?

Reply:
Yes! Great that you bring this up. I think we can show a warning, hopefully we have enough styles already. Let's display this when the address is being entered, but not afterwards?

Reply:
<img width="528" alt="Screenshot 2023-10-18 at 20 15 28" src="https://github.com/filecoin-station/desktop/assets/10247/1c43a416-7dbf-4b0a-9e8d-5e3d727bcbfc">

Like this?

Reply:
<img width="519" alt="Screenshot 2023-10-18 at 20 15 54" src="https://github.com/filecoin-station/desktop/assets/10247/a37ec172-b429-4930-9a8f-26627c41f86e">

Like this?

Reply:
Nice! 🤩